processing/data_collection/gazette/spiders/rj_niteroi.py

```diff
@@ -0,0 +1,34 @@
+from gazette.items import Gazette
+import datetime as dt
+import scrapy
+
+
+class RjNiteroiSpider(scrapy.Spider):
```

Comment:
Please extend you spider from  `BaseGazetteSpider`
https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/spiders/base.py.